ui/src/environments/environment.prod.ts

```diff
@@ -4,6 +4,7 @@ export const environment: Environment = {
   displayTag: '',
   shouldShowDisplayTag: false,
   allOfUsApiUrl: 'https://api.workbench.researchallofus.org',
+  captchaSiteKey: '',
```

Comment:
If you're not going to enable this straight away, please file a ticket to enable captcha in prod.